frontend/lib/src/components/widgets/NumberInput/NumberInput.test.tsx

```diff
@@ -452,5 +452,12 @@ describe("NumberInput widget", () => {
       expect(wrapper.state("value")).toBe(2)
       expect(stepUpButton(wrapper).prop("disabled")).toBe(true)
     })
+
+    it("hides stepUp and stepDown buttons when width is smaller than 120px", () => {
+      const props = getIntProps({ default: 1, step: 1, max: 2, hasMax: true })
+      const wrapper = shallow(<NumberInput {...props} width={100} />)
+
+      expect(wrapper.find("button").exists()).toBe(false)
+    })
```

Comment:
Could you add the converse test - "shows stepUp and stepDown buttons when width is greater than 120px"

Reply:
Sure thing! Added on [95b0381](https://github.com/streamlit/streamlit/pull/7050/commits/95b038195ea6e4b474252fa5dc83bd102bcd26b1)